src/MomentRe.re

```diff
@@ -297,12 +297,18 @@ external momentWithFormat: (string, string) => Moment.t = "moment";
 [@bs.module] external momentWithDate: Js.Date.t => Moment.t = "moment";
 
 [@bs.module]
-external momentWithFormats: (string, list(string)) => Moment.t = "moment";
+external momentWithFormats: (string, array(string)) => Moment.t = "moment";
```

Comment:
array(string), not list(string). Because list(string) not convert properli to js array, but reason array convert good so its bug. This change is little dangerous because someone theoreticly can used it with list, but i checked that if try to use it with list it cause runetime error so I think no one used it.